### PR TITLE
fix: question item point not refreshing

### PR DESCRIPTION
**Pull request by stevenao**

### CODE REVIEW

File: components/activity/editor/collection/custom/quiz/d2l-activity-collection-item-quiz.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const componentClass = class extends HypermediaStateMixin(ListItemLinkMixin(LitE
 	render() {
 		return this._renderListItem({
 			//${guard([this._activityHref, this.token], () => html`<d2l-activity-list-item-content href="${this._activityHref}" .token="${this.token}"></d2l-activity-list-item-content>`)}`
-			content: html`${guard([this._activityHref, this.token], () => html`
+			content: html`${guard([this._activityHref, this.token, this._points], () => html`
```
Comment (me-louie) [inline components/activity/editor/collection/custom/quiz/d2l-activity-collection-item-quiz.js:43]: I'm wondering, will Sections automatically refresh on edit as well?

Comment (stevenao) [reply components/activity/editor/collection/custom/quiz/d2l-activity-collection-item-quiz.js:43]: href="https://618b5cb9-170f-4a55-a745-3374f317f55b.activities.api.proddev.d2l/activities/6606_qsection_172/usages/6609"
the section domain api doesn't change after it is updated.. so, I am not 100% sure. We will have to test it out in this case..

Comment (me-louie) [reply components/activity/editor/collection/custom/quiz/d2l-activity-collection-item-quiz.js:43]: It didn't seem to work when I was testing with my LMS Sections changes so we'll probably have to look into it more. This is great for points though.

**Review by me-louie — APPROVED** [review]
LGTM

**Pull request merged**